custom_components/rct_power/coordinator.py

```diff
@@ -27,10 +27,10 @@ def __init__(
         hass: HomeAssistant,
         entry: ConfigEntry,
         *,
-        name_suffix: str,
         client: RctPowerApiClient,
+        name_suffix: str,
         object_ids: list[int],
-        update_interval: timedelta | None = None,
+        update_interval: int,
```

Comment:
Since we're losing the unit information that was previously encapsulated in the time delta, how about we document it here?

```suggestion
        update_interval: int,  # update interval in seconds
```